### DISCUSSION TITLE
タグミュート機能を使用しているがミュートされていない

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#368] [要望・アイデア・不具合]
[タグミュート機能を正式にリリースします](https://blog.qiita.com/release-tag-mute-general/?utm_source=qiita&utm_content=official_column&utm_medium=referral)より、タグミュート機能の説明について以下の記載があります。

> タグミュート機能を利用すると、特定のタグがついたコンテンツを非表示にすることができます。
[ホーム](https://qiita.com/)
[タイムライン](https://qiita.com/timeline)
[トレンド](https://qiita.com/trend)
検索ページ (例: [「Markdown記法 チートシート」の検索結果](https://qiita.com/search?q=Markdown%E8%A8%98%E6%B3%95+%E3%83%81%E3%83%BC%E3%83%88%E3%82%B7%E3%83%BC%E3%83%88))

私は「ポエム」のタグをミュートしています。
しかし、添付の通りにトレンドに、ミュートしたタグが表示されているのは何故でしょうか？
![スクリーンショット 2022-12-05 20 59 25](https://user-images.githubusercontent.com/38848514/205632635-a14ea877-968b-4bf3-b1ce-02a02b795bc3.png)
![スクリーンショット 2022-12-05 20 59 30](https://user-images.githubusercontent.com/38848514/205632652-76df894f-290c-44a2-9b2f-fabaf852e03a.png)

**Comment by kyntk** [comment]
@Brutus03 タグミュート機能についてですが、

> 特定のタグがついたコンテンツを非表示

とあるように、「タグ」自体をミュートするのではなく、「タグがついたコンテンツ」をミュートする機能となります 🙇 💦 
この「コンテンツ」は現在だと記事とコメントを指していますが、今後は質問や回答についても想定しております。

そのため、ランキングからミュートをしたタグ自体は非表示にしていません。

**Reply by Brutus03** [reply]
@kyntk 
> とあるように、「タグ」自体をミュートするのではなく、「タグがついたコンテンツ」をミュートする機能となります 🙇 💦

回答について承知しました。
また改めてタグ機能の仕様についても理解及び動作確認できました。

逆説的ではありますが、タグ自体をミュートすると、そもそもランキングの意味も薄れそうですしね。
回答について納得したので、本件はクローズで大丈夫です。
ありがとうございました。
